src/test/goldens/polymer/expected.d.ts

```diff
@@ -819,7 +819,7 @@ declare namespace Polymer {
      * function should match the sort function passed to `Array.sort`.
      * Using a sort function has no effect on the underlying `items` array.
      */
-    sort: Function|null;
```

Comment:
Function is legal TypeScript, and it has the same semantics AFAIK as Closure's, why not just keep it that way?